<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameTents</class>
 <widget class="QFrame" name="FrameTents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Añadir cliente</string>
        </property>
        <property name="icon">
         <iconset resource="default.qrc">
          <normaloff>:/imgs/add.svg</normaloff>:/imgs/add.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxDorms">
        <property name="text">
         <string>Mostrar dormis</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mostrar:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboClientStatus">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Todos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Huéspedes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vencidos</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="list">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeWidget::item {
	margin-top: 18px;
	margin-right: 10px;
}</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de entrada</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de salida</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ubicación</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Per/Carp</string>
      </property>
      <property name="toolTip">
       <string>Personas/Carpas</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Seña</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vehículos</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionListEdit">
   <property name="text">
    <string>Editar</string>
   </property>
  </action>
  <action name="actionListPrint">
   <property name="text">
    <string>Imprimir comprobante</string>
   </property>
  </action>
  <action name="actionListDelete">
   <property name="text">
    <string>Borrar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="default.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>FrameTents</receiver>
   <slot>onAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboClientStatus</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrameTents</receiver>
   <slot>requestRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDorms</sender>
   <signal>clicked()</signal>
   <receiver>FrameTents</receiver>
   <slot>requestRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddClicked()</slot>
  <slot>refreshData()</slot>
  <slot>requestRefresh()</slot>
 </slots>
</ui>
